Makefile

```diff
@@ -170,12 +170,13 @@ BEATS_MODULE=$(shell go list -m -f {{.Path}} all | grep github.com/elastic/beats
 
 .PHONY: update-beats
 update-beats: update-beats-module update
-	@echo --- Use this commit message: Update beats framework to \
-		$(shell go list -m -f {{.Version}} $(BEATS_MODULE) | cut -d- -f3)
+	@echo --- Use this commit message: Update to elastic/beats@$(shell go list -m -f {{.Version}} $(BEATS_MODULE) | cut -d- -f3)
 
 .PHONY: update-beats-module
 update-beats-module:
 	go get -d -u $(BEATS_MODULE)@$(BEATS_VERSION)
+	diff -u .go-version $$(go list -m -f {{.Dir}} $(BEATS_MODULE))/.go-version \
```

Comment:
Unrelated, but I usually also run a `go mod tidy` manually after the beats update. Since you are on it, would you mind adding that?